src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -324,12 +340,26 @@ double BaseCouplingScheme::getTimeWindowSize() const
   return _timeWindowSize;
 }
 
+double BaseCouplingScheme::getWindowStartTime() const
+{
+  return _time - _computedTimeWindowPart;
+}
+
+double BaseCouplingScheme::getNormalizedWindowTime() const
+{
+  const double timeWindowStart        = getWindowStartTime();
+  const double timeWindowSize         = getTimeWindowSize();
+  const double computedTimeWindowPart = getTime() - timeWindowStart;
+  // const double computedTimeWindowPart = getComputedTimeWindowPart();  // @todo make public?
```

Comment:
?

Reply:
Cleaned this up: 1eb09ea16d641903a8d70da0b7f9701dd338f7b5